### DISCUSSION TITLE
2023-07-12

### DISCUSSION THREAD

**Discussion by yeopto** [woowacourse-teams/2023-haru-study#20] [daily_meeting]
### 일정 있는 사람
- 노아(근로) 17:00시

### 오늘의 할 일
- 공통
    - ~~브랜치 전략~~
    - api 명세 픽스
    - 세부정책 포함
- 백엔드
    - 기능목록 작성
- 프론트엔드
    - 디자인

### 내일의 할 일

- 공통
  - API 명세 픽스(세부정책 포함)
- 백엔드
  - 기능목록 작성
- 프론트엔드
  - 디자인
  - 기능 명세 

### 언젠가 해야할 일
- 기획 변경된 이유 정리

**Comment by nlom0218** [comment]
### 체크인

10점 - 밝게 활기차게 :)


### 체크아웃

10점 - 딱히 마이너스 할 것이 없다!

**Comment by yeopto** [comment]
### 체크인

6점 - 너무 피곤하네요 오늘 :(

### 체크아웃

7점 - 어늘 하루도 무사히 끝

**Comment by aak2075** [comment]
### 체크인

10점 - 기분 좋게 하루를 시작해요

### 체크아웃

9점 - 생각보다 오늘 진행한 일들이 오래 걸렸다.

**Comment by jaehee329** [comment]
### 체크인
6점 - 많이 못 잤어요
### 체크아웃
5점 - 회의를 많이 했어요

**Comment by woosung1223** [comment]
### 체크인
10점 - 점수 까일 이유가 없어서 10점

### 체크아웃
7점 - 기획은 힘들어서 7점 ㅠㅠ

**Comment by woo-jk** [comment]
### 체크인
8점 - 피곤해서 -1 빨래 잘못해서 옷에서 냄새남 -1

### 체크아웃
8점 - 피곤해서😅 -2

**Reply by woosung1223** [reply]
맡으러가도되나요?
